apps/resume-web/src/routes/technical-skills/-src/components/Form.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, LabeledTextInput, LabledNativeSelect } from "@resume/ui"
+import { ComponentProps } from "react"
+import { useFormStatus } from "react-dom"
+
+
+const FormButton = () => {
+  const {
+    pending
+  } = useFormStatus();
+  return <Button disabled={pending} className='py-6' themeColor='primary' variant='solid'>
+    Next
+  </Button>
+}
+
+export interface TechnicalSkills { }
+
+export type TechnicalSkillsFormProps = TechnicalSkills & ComponentProps<'form'>;
+
+const options = [
+  {
+    label: 'React',
+    value: 'react'
+  },
+  {
+    label: 'Vue',
+    value: 'vue'
+  },
+  {
+    label: "Angular",
+    value: 'angular'
+  }
+]
```

Comment:
Options should be paramtrized, since is possibel that this going to come from a backend